packages/earthlyls/package.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: earthlyls
+description: |
+  A fast language server for earthly.
+homepage: https://github.com/glehmann/earthly
+licenses:
+  - MIT
+languages:
+  - Earthly
+categories:
+  - LSP
+
+source:
+  # renovate:versioning=loose
```

Comment:
The project seems to apply semver versioning, in which case we don't need to override the default versioning strategy.
```suggestion
```